Src/PerformanceCollector/Shared/Implementation/IPerformanceCounterData.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
```

Comment:
I am considering to remove this interface completely, and PerformanceCounterData will be consistent for different Collectors.